ctl/waypoint/waypoint.go

```diff
@@ -37,7 +37,8 @@ import (
 	"istio.io/istio/pkg/config/constants"
 	"istio.io/istio/pkg/config/protocol"
 	"istio.io/istio/pkg/config/schema/gvk"
-	"istio.io/istio/pkg/kube"
+	"kmesh.net/kmesh/pkg/kube"
```

Comment:
this is a golint error

Reply:
> this is a golint error

Sorry, I can't see where the issue is. Could you help point it out specifically?

Reply:
In go, the order of packages introduced is standard pkg, tripartite pkg and packages within the organisation, with each partition split by blank lines.

You have now use `kmesh.net/kmesh/pkg/kube`, referencing a pkg within the organisation. It should be placed in the third partition